.github/workflows/pythonlinters.yml

```diff
@@ -33,6 +33,7 @@ jobs:
         run: |
           ./lint-flake8.sh
 
+      # note: pyre is run informationally, never to fail the build
```

Comment:
Pyre actually does fail the build.  pylint and mypy are run with `|| :` at the end of their shell command which cause them to not fail the build even if the run turned up errors.

Reply:
My bad, I focused on the key 'if' and did not see this...